Activities/Cryptography/UiPath.Cryptography.Activities/KeyedHashFile.cs

```diff
@@ -154,6 +165,14 @@ protected override string Execute(CodeActivityContext context)
                     filePath = localFile.LocalPath;
                 }
 
+                if (keyEncoding == null)
```

Comment:
Shouldn't the helper function KeyEncodingOrString be applied here?

Reply:
done, thanks for spot